web/tsconfig.json

```diff
@@ -26,7 +26,7 @@
         "./src/*"
       ]
     },
-    "target": "ES2017"
```

Comment:
This isn't neccisarily part of upgrading to tailwind, but I noticed this while playing with your pr. all the browsers we support support at lest es2022 so you can bump this so you can get access to more stuff like Array.at or whatever